packages/app/src/navigation/SidebarNavigationHeader.vue

```diff
@@ -71,6 +70,7 @@ const showModal = ref(false)
 
 const props = defineProps<{
   gql: SidebarNavigationHeaderFragment
+  isNavBarExpanded: boolean
```

Comment:
Still new to GraphQL & this setup, but aren't you able to pull this value off of the gql prop since the `isSideNavigationOpen` be updated when this is toggled in SideNavigation?

Reply:
Yeah, it can be, the issue was that b/c there are views that do not allow to have the sideNav open, we had to do [this](https://github.com/cypress-io/cypress/pull/20026/files/e881adfbad91e22e521549c3c7413d39766f3fd6), so we do not show it open based on the user preferences